avocado/core/safeloader.py

```diff
@@ -497,8 +497,8 @@ def _examine_class(path, class_name, match, target_module, target_class,
 
             modules_paths = [parent_path,
                              os.path.dirname(module.path)] + sys.path
-            _, found_ppath, _ = imp.find_module(parent_module,
-                                                modules_paths)
+            found_ppath = importlib.machinery.PathFinder.find_spec(
```

Comment:
Since your are using only Pathfinder, my suggestion is to change the import to:

```python
from importlib.machinery import PathFinder
```

So this line will be shorter:

```python
found_ppath = PathFinder.find_spec(...
```

Reply:
Sure @beraldoleal, that`s a good point. I will change it!